scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -235,6 +235,8 @@ def _create_command(self, robot_test):
             command = [self.generator_cmd, "run_renode_test"]
         elif self.call_make_run:
             command = [self.generator_cmd, "run"]
+        elif self.instance.testsuite.type == "unit":
+            command = [self.binary]
```

Comment:
nit: you can think about setting here the binary path for unit test - moving code from 
https://github.com/zephyrproject-rtos/zephyr/blob/aabf130f98deaac239a5d1977f2f26b0f96f544f/scripts/pylib/twister/twisterlib/testinstance.py#L212